src/c-writer.cc

```diff
@@ -2783,7 +2824,11 @@ void CWriter::WriteParamTypes(const FuncDeclaration& decl) {
       } else {
         needs_comma = true;
       }
-      Write(decl.GetParamType(i));
+      if (is_varargs && i == decl.GetNumParams() - 1) {
```

Comment:
nit: `()` for `i == ...`

Reply:
I'm not sure I understand this comment...

Reply:
Oh, got it now.  You want me to put the second condition in parentheses... :)

Reply:
Done.